File: 2.js
const readline = require("readline");
const { json } = require("stream/consumers");
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
});
rl.question("Enter a letter: ", function(x) {
  if (x.length === 1 && x === x.toUpperCase() && x !== x.toLowerCase()) {
    console.log("uppercase.");
  } else {
    console.log("not uppercase.");
  }
  rl.close();
});

File: 2.cpp
#include <iostream>
#include <ctype.h>
using namespace std;

int main() 
{
    char x;
    cout << "Enter a letter: ";
    cin >> x;
    if (isupper(x)) 
    {
        cout << "uppercase." << endl;
    }
    else 
    {
        cout << "not uppercase." << endl;
    }
    return 0;
}